bokeh/models/tests/test_transforms.py

```diff
@@ -0,0 +1,133 @@
+from textwrap import dedent
+
+import pytest
+
+from bokeh.models import CustomJSTransform, Slider
+
+flexx = pytest.importorskip("flexx")
+
+def test_customjstransform_from_py_func_no_args():
+
+    def cosine():
+        from flexx.pyscript import window
+        return window.Math.cos(x) # noqa
+
+    def v_cosine():
+        from flexx.pyscript import window
+        return [window.Math.cos(x) for x in xs] # noqa
```

Comment:
I am fairly down on "python->js transpilation" in general at this point. So, I agree this can be lived with as is for now.